joomla/joomla-cms
# Article module fix read more title

Pull Request for Issue #44135 

### Summary of Changes
Fix the read more title functionality.

### Testing Instructions
- Toggle the function Read More with Title on and off and see if the title is shown or hidden
- Change the Read More Limit (characters) to 15 to see if the title is shortened,

### Actual result BEFORE applying this Pull Request
Toggle Read More with Title not working

### Expected result AFTER applying this Pull Request
Toggle Read More with Title is working and truncation of title is working

### Link to documentations
Please select:
